firefox_ui_tests/remote/security/test_dv_certificate.py

```diff
@@ -0,0 +1,103 @@
+        FirefoxTestCase.tearDown(self)
+
+    @skip_under_xvfb
+    def test_larry_blue(self):
+
+        with self.marionette.using_context('content'):
+            self.marionette.navigate(self.test_url)
+
+        self.cert = self.browser.tabbar.selected_tab.certificate
+
+        favicon_hidden = self.marionette.execute_script("""
+          return arguments[0].hasAttribute("hidden");
+        """, script_args=[self.locationbar.favicon])
+
+        self.assertFalse(favicon_hidden)
+
+        self.assertIn('identity-icons-https',
+                      self.locationbar.favicon.value_of_css_property('list-style-image'),
+                      'There is a lock icon')
+
+        self.assertEqual(self.identity_popup.box.get_attribute('className'),
+                         'verifiedDomain',
+                         'Identity is verified')
+
+        panel = self.identity_popup.popup
+        self.identity_popup.box.click()
+        self.wait_for_condition(lambda _: self.identity_popup.is_open)
+
+        doorhanger_class = self.identity_popup.popup.get_attribute('className')
+        self.assertEqual(doorhanger_class,
+                         'verifiedDomain',
+                         'The Larry UI is domain verified (aka Blue)')
+
+        encryption_icon = self.identity_popup.encryption_icon
+        lock_image = encryption_icon.value_of_css_property('list-style-image')
+        self.assertNotEqual(lock_image, 'none', 'There is a lock icon')
+
+        # Bug 443116
+        # Larry strips the 'www.' from the CName using the eTLDService
+        # This is expected behaviour for the time being
+        host_name = self.identity_popup.host.get_attribute('textContent')
+        expected_name = self.marionette.execute_script("""
+           let Cu = Components.utils;
+           Cu.import("resource://gre/modules/Services.jsm");
+           return Services.eTLD.getBaseDomainFromHost(arguments[0]);
+        """, script_args=[self.cert['commonName']])
+
+        self.assertEqual(host_name, expected_name,
+                         'The site identifier string is equal to the cert host')
+
+        verifier_label = self.browser.get_property('identity.identified.verifier')
+        verifier_label = verifier_label.replace("%S", self.cert['issuerOrganization'])
+        verifier_element = self.identity_popup.verifier
+
+        self.assertEqual(verifier_element.get_attribute('textContent'),
+                         verifier_label,
+                         'The "Verified by: %S" string is set')
+
+        def opener(mn):
+            self.identity_popup.more_info_button.click()
+
+        page_info_window = self.browser.open_page_info_window(opener)
+        deck = page_info_window.deck
+
+        self.assertEqual(deck.selected_panel, deck.security,
+                         "The security tab is selected by default")
+
+        web_id_domain = deck.security.domain.get_attribute('value')
+        self.assertEqual(web_id_domain, self.cert['commonName'],
```

Comment:
Also why did you strip out the replacements in this string? Is that not necessary?